governance/program/src/state/proposal.rs

```diff
@@ -474,12 +527,15 @@ impl ProposalV2 {
                 .unwrap();
 
         if yes_vote_weight >= min_vote_threshold_weight
-            && yes_vote_weight > (max_vote_weight - yes_vote_weight)
+            && yes_vote_weight > (max_vote_weight.checked_sub(yes_vote_weight).unwrap())
```

Comment:
nit: For all these, you can do a `saturating_sub` here for the check and that way you're still ok with a bad `max_vote_weight` that's too low.  Since you're coercing elsewhere, that would line up with the idea that it's ok for the `max_vote_weight` to be wrong.

Reply:
Done